pkg/cli/cli.go

```diff
@@ -569,18 +569,23 @@ func runCreateCVDCommand(c *cobra.Command, flags *CreateCVDFlags, opts *subComma
 		flags.CreateCVDOpts.Host = ins.Name
 	}
 	statePrinter.Print(createCVDStateMsg)
-	cvd, err := createCVD(service, *flags.CreateCVDOpts)
+	cvds, err := createCVD(service, *flags.CreateCVDOpts)
 	statePrinter.PrintDone(createCVDStateMsg)
 	if err != nil {
 		return err
 	}
-	statePrinter.Print(connectCVDStateMsg)
-	cvd.ConnStatus, err = ConnectDevice(flags.CreateCVDOpts.Host, cvd.Name, &command{c, &flags.Verbose}, opts)
-	statePrinter.PrintDone(connectCVDStateMsg)
-	if err != nil {
-		err = fmt.Errorf("Failed to connect to device: %w", err)
+	if len(cvds) > 0 {
+		cvd := cvds[0]
```

Comment:
why connect only the first device? why not all?

Reply:
connected to all, wanted your feedback here.